mappings/net/minecraft/client/util/telemetry/TelemetryManager.mapping

```diff
@@ -0,0 +1,44 @@
+CLASS net/minecraft/unmapped/C_wwbyxkgc net/minecraft/client/util/telemetry/TelemetryManager
+	FIELD f_dgegipht session Lcom/mojang/authlib/minecraft/TelemetrySession;
+	FIELD f_dkujvsyq client Lnet/minecraft/unmapped/C_diboefya;
+	FIELD f_egikwkbj TELEMETRY_SENDER_COUNTER Ljava/util/concurrent/atomic/AtomicInteger;
+	FIELD f_igtmozun gameModeData Lnet/minecraft/unmapped/C_wwbyxkgc$C_fsjxjpon;
+	FIELD f_qmsnnjid serverBrand Ljava/lang/String;
+	FIELD f_qpilocfm sent Z
+	FIELD f_ywwymtzy TELEMETRY_SENDER_EXECUTOR Ljava/util/concurrent/Executor;
+	METHOD <init> (Lnet/minecraft/unmapped/C_diboefya;Lcom/mojang/authlib/minecraft/UserApiService;Ljava/util/Optional;Ljava/util/Optional;Ljava/util/UUID;)V
+		ARG 1 client
+		ARG 2 userApiService
+		ARG 3 userId
+		ARG 4 clientId
+		ARG 5 deviceSessionId
+	METHOD m_aiwjwbjb (Ljava/lang/Runnable;)Ljava/lang/Thread;
+		ARG 0 action
+	METHOD m_ajbqkwpi setServerBrandAndSend (Ljava/lang/String;)V
```

Comment:
Why not just `setServerBrand`?

Reply:
It also executes `TelemetryManager#send(GameModeData)`

Reply:
I haven't read the conventions yet but would something like `updateServerBrand` fit? A bit less wordy and still descriptive